dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSF22StatelessViewTests.java

```diff
@@ -57,13 +59,15 @@ public static void setup() throws Exception {
         ShrinkHelper.exportToServer(server, "dropins", mojarraApp);
         server.addInstalledAppForValidation(MOJARRA_APP);
 
-        WebArchive myfacesApp = ShrinkWrap.create(WebArchive.class, MYFACES_APP + ".war")
-                        .addPackage("jsf.view.beans");
-        myfacesApp = FATSuite.addMyFaces(myfacesApp);
-        myfacesApp = (WebArchive) ShrinkHelper.addDirectory(myfacesApp, "publish/files/permissions");
-        myfacesApp = (WebArchive) ShrinkHelper.addDirectory(myfacesApp, "test-applications/" + MOJARRA_APP + "/resources");
-        ShrinkHelper.exportToServer(server, "dropins", myfacesApp);
-        server.addInstalledAppForValidation(MYFACES_APP);
+        if(!JakartaEE10Action.isActive()){
+            WebArchive myfacesApp = ShrinkWrap.create(WebArchive.class, MYFACES_APP + ".war")
+                            .addPackage("jsf.view.beans");
+            myfacesApp = FATSuite.addMyFaces(myfacesApp);
+            myfacesApp = (WebArchive) ShrinkHelper.addDirectory(myfacesApp, "publish/files/permissions");
+            myfacesApp = (WebArchive) ShrinkHelper.addDirectory(myfacesApp, "test-applications/" + MOJARRA_APP + "/resources");
+            ShrinkHelper.exportToServer(server, "dropins", myfacesApp);
+            server.addInstalledAppForValidation(MYFACES_APP);
+        }
```

Comment:
With the current changes, I tried to have only the working Mojarra tests running.The MyFaces 4.0.0-RC1 apps encounter errors during start up due to the CDI Injection / Producer problem, so I just thought it would be better deploy the app.